### PR TITLE
fix: log.Fatal will call os.Exit, use log.Println instead

### PR DESCRIPTION
**Pull request by Abirdcfly**
Signed-off-by: Abirdcfly <fp544037857@gmail.com>

## This PR

- use `log.Println` instead of `log.Fatal` to avoid immediate exit.
 

### Related Issues

Fixes #

### Notes

`log.Fatal` source code see https://cs.opensource.google/go/go/+/refs/tags/go1.18.4:src/log/log.go;l=364-366;drc=2580d0e08d5e9f979b943758d3c49877fb2324cb

[Description](https://pkg.go.dev/log#Fatal) of `log.Fatal`: Fatal is equivalent to Print() followed by a call to os.Exit(1).

### CODE REVIEW

**Review by thisthat — APPROVED** [review]

File: helm-service/main.go [hunk 1]
```diff
@@ -31,7 +29,7 @@ import (
 	keptncommon "github.com/keptn/go-utils/pkg/lib/keptn"
 	keptnv2 "github.com/keptn/go-utils/pkg/lib/v0_2_0"
 
-	utils "github.com/keptn/go-utils/pkg/api/utils"
+	"github.com/keptn/go-utils/pkg/api/utils"
```
Comment (thisthat) [inline helm-service/main.go:32]: please, revert this change

**Review by thisthat — DISMISSED** [review]

File: helm-service/main.go (hunk 1 above)
Comment (Abirdcfly) [reply helm-service/main.go]: revert it.

**Review by odubajDT — APPROVED** [review]
